README.md

```diff
@@ -35,20 +36,73 @@ value = phin.getData(
 ```python
 #Sample value from getData()
 {
-	"waterData":
-		{"TA":80,"CYA":60,"TH":450, "temperature": 80.2,
-		"status":"balanced"}
+  'pool': {
+    'status_title': 'needs-attention',
+    'status_id': 2
+  },
+  'waterData': {
+    'ta': 80,
+    'cya': 60,
+    'th': 450,
+    'temperature': 75.0,
+    'ph': {
+      'value': 7.2,
+      'status': 3
+    },
+    'orp': {
+      'value': 550.2,
+      'status': 2
+    }
+  },
+  'vesselData': {
+    'battery': {
+      'value': 3000.2,
+      'status': 3
+    },
+    'rssi': {
+      'value': -92,
+      'status': 3
+    }
+  }
 }
 ```
 
 
 ## Implemented Data Types
 
 ```python
-"TH" #Total Hardness (Nominal 150-399 ppm)
-"CYA" #Cyanuric Acid (Nominal 20-99 ppm)
-"TA" #Total Alkalinity (Nominal 80-150 ppm)
+pool:
+"status_title" #String of Overall Status of Pool
+"status_id" #Indexed status value (See Key Below)
 
+waterData:
+"th" #Total Hardness (Nominal 150-399 ppm)
+"cya" #Cyanuric Acid (Nominal 20-99 ppm)
+"ta" #Total Alkalinity (Nominal 80-150 ppm)
 "temperature" #Temperature in Fahrenheit
-"status" #Overall Status of Water
-``` 
+"ph":
+	"value" #Value of pH
+	"status" #Status of pH (Integer 1-5)
+"orp":
+	"value" #Value of ORP in mV
+	"status" #Status of ORP (1,3 or 5)
+
+vesselData:
+"rssi"
+	"value" #Value of RSSI in dB
+	"status" #Status of RSSI (1,3 or 5)
+"battery":
+	"value" #Value of battery in mV
+	"status" #Status of Battery (1,3 or 5)
```

Comment:
why not support a battery status of like 0 - 100%?

Reply:
Well, it might still function at "0%" so its not completely accurate, people will think like, oh 1% itll die any minute. But it might last for much longer.